packages/contracts-bedrock/src/L1/OPContractsManager.sol

```diff
@@ -621,28 +700,30 @@ contract OPContractsManagerUpgrader is OPContractsManagerBase {
     /// @notice Deploys and sets a new dispute game implementation
     /// @param _l2ChainId The L2 chain ID
     /// @param _disputeGame The current dispute game implementation
+    /// @param _newAnchorStateRegistryProxy The new anchor state registry proxy
     /// @param _gameType The type of game to deploy
     /// @param _opChainConfig The OP chain configuration
-    /// @param _blueprints The blueprint addresses
-    /// @param _implementations The implementation addresses
     /// @param _opChainAddrs The OP chain addresses
     function deployAndSetNewGameImpl(
         uint256 _l2ChainId,
         IDisputeGame _disputeGame,
+        IAnchorStateRegistry _newAnchorStateRegistryProxy,
         GameType _gameType,
         OPContractsManager.OpChainConfig memory _opChainConfig,
-        OPContractsManager.Blueprints memory _blueprints,
-        OPContractsManager.Implementations memory _implementations,
         ISystemConfig.Addresses memory _opChainAddrs
     )
         internal
     {
+        OPContractsManager.Blueprints memory bps = getBlueprints();
```

Comment:
nit, do we have a convention that internal/private methods need leading underscores? Didn't see it in the style guide, and we are not consistent in the codebase, but this is fine for now and something to fix with a semgrep rule